packages/iTwinUI-react/src/core/Table/columns/actionColumn.tsx

```diff
@@ -34,9 +42,9 @@ const ACTION_CELL_ID = 'iui-table-action';
  *   ),
  * },
  */
-export const ActionColumn = <T extends Record<string, unknown>>({
-  columnManager = false,
-} = {}) => {
+export const ActionColumn = <T extends Record<string, unknown>>(
+  { columnManager }: ActionColumnProps = { columnManager: false },
```

Comment:
`{columnManager = false}: ActionColumnProps`

Reply:
I actually was trying this way because I was getting the error on places where just `ActionColumn()` was used that it expected 1 argument but got 0. That is why I tried changing it to the code I have currently to avoid that error.

Reply:
**[DONE]** Thanks. I reverted back line 46 to the code you recommended.